netket/vqs/mc/mc_state/state.py

```diff
@@ -214,6 +248,9 @@ def __init__(
 
         self.n_discard_per_chain = n_discard_per_chain
 
+        if chunk_size is None:
+            self.chunk_size = chunk_size
```

Comment:
Why do we not set non-None chunk sizes?
```suggestion
        self.chunk_size = chunk_size
```

Reply:
Not really a reason. It's already set to None, and I preferred not to set stuff unless really needed

Reply:
Umm, this is in the constructor, this is the only place where chunk_size would be set if you pass it to the constructor...